examples/spring-boot-jetty/src/main/java/example/springframework/boot/jetty/HelloConfiguration.java

```diff
@@ -0,0 +1,102 @@
+package example.springframework.boot.jetty;
+
+import org.eclipse.jetty.server.Server;
+import org.eclipse.jetty.util.Loader;
+import org.eclipse.jetty.webapp.WebAppContext;
+import org.springframework.beans.factory.ObjectProvider;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnClass;
+import org.springframework.boot.web.embedded.jetty.JettyServerCustomizer;
+import org.springframework.boot.web.embedded.jetty.JettyServletWebServerFactory;
+import org.springframework.boot.web.embedded.jetty.JettyWebServer;
+import org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import com.linecorp.armeria.server.ServerBuilder;
+import com.linecorp.armeria.server.healthcheck.HealthChecker;
+import com.linecorp.armeria.server.jetty.JettyService;
+import com.linecorp.armeria.spring.ArmeriaServerConfigurator;
+
+import jakarta.servlet.Servlet;
+
+/**
+ * Configures an Armeria server to redirect the incoming requests to the Jetty instance provided by
+ * Spring Boot. It also sets up a {@link HealthChecker} so that it works well with a load balancer.
+ */
+@Configuration
+public class HelloConfiguration {
+
+    /**
+     * Returns a new {@link HealthChecker} that marks the server as unhealthy when Tomcat becomes unavailable.
+     */
+    @Bean
+    public HealthChecker jettyHealthChecker(ServletWebServerApplicationContext applicationContext) {
+        final Server server = jettyServer(applicationContext).getServer();
+        return server::isRunning;
+    }
+
+    /**
+     * Returns a new {@link JettyService} that redirects the incoming requests to the Jetty instance
+     * provided by Spring Boot.
+     */
+    @Bean
+    public JettyService jettyService(ServletWebServerApplicationContext applicationContext) {
+        final JettyWebServer jettyWebServer = jettyServer(applicationContext);
+        assert jettyWebServer instanceof JettyWebServerWrapper;
+        final JettyWebServerWrapper webServerWrapper = (JettyWebServerWrapper) jettyWebServer;
+        return JettyService.of(jettyWebServer.getServer(), null, false,
+                               webServerWrapper::doStart, webServerWrapper::stop);
```

Comment:
IIUC, this is the same with `JettyService.of(jettyWebServer.getServer(), null, false)`.
Could you explain why we need a separate start and stop tasks apart from `jetty.server.Server.start()` and `jetty.server.Server.stop()`

Reply:
IIUC, `JettyWebServerWrapper.start()` performs but doesn't `JettyWebServerWrapper.getServer().start()` starts the Jetty server?
That means we can just pass the unwrapped `jettyWebServer.getServer()` to start the Jetty server by Armeria?

Reply:
I realized that we need to start `JettyWebServer` instead of `jetty.server.Server`.

Reply:
Let Spring Boot manages the lifecycle and rollback `JettyService` to the original form. Thanks! 😉